workitem/link/link_repository.go

```diff
@@ -458,3 +459,30 @@ func (r *GormWorkItemLinkRepository) WorkItemHasChildren(ctx context.Context, pa
 	}
 	return hasChildren, nil
 }
+
+// GetParentID returns parent ID of the given work item if any
+func (r *GormWorkItemLinkRepository) GetParentID(ctx context.Context, ID uuid.UUID) (*uuid.UUID, error) {
```

Comment:
A test would be good.

Reply:
Yes Thanks, added in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/42113cf08b6e21d2ed4fca6e22eaabf6e39cb294